src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $allowMultipleNulls = true;
+        if (isset($options['allowMultipleNulls'])) {
```

Comment:
I think those strings should not be magic strings but class constants.
This way they can be used from outside without the risk of typos.
We should be doing this more often in general with those option keys available per class IMO.